### PR TITLE
Add cpu_count, cpu_percent, cpus parameters.

### PR DESCRIPTION
**Pull request by AlexeyRokhin**
Fixes https://github.com/docker/compose/issues/4582
Uses docker-py 2.3.0 to support these parameters.

### CODE REVIEW

File: compose/config/config_schema_v2.2.json [hunk 1]
```diff
@@ -74,8 +74,11 @@
           ]
         },
         "container_name": {"type": "string"},
+        "cpu_count": {"type": "integer", "minimum": 0},
+        "cpu_percent": {"type": "integer", "minimum": 0, "maximum": 100},
         "cpu_shares": {"type": ["number", "string"]},
         "cpu_quota": {"type": ["number", "string"]},
+        "cpus": {"type": ["number", "string"], "minimum": 0},
```
Comment (shin-) [inline compose/config/config_schema_v2.2.json]: I don't think "string" is a valid type here.

File: compose/service.py [hunk 2]
```diff
@@ -798,6 +801,10 @@ def _get_container_host_config(self, override_options, one_off=False):
             init_path = options.get('init')
             options['init'] = True
 
+        nano_cpus = None
+        if 'cpus' in options:
+            nano_cpus = int(options.get('cpus') * 1000000000)
```
Comment (shin-) [inline compose/service.py]: We should check in `compose/config/validation.py` that the result of `cpus * 10^9` is not a decimal value, [same as here](https://github.com/moby/moby/blob/v17.05.0-ce/opts/opts.go#L378-L381)

File: compose/service.py [hunk 3]
```diff
@@ -798,6 +801,13 @@ def _get_container_host_config(self, override_options, one_off=False):
             init_path = options.get('init')
             options['init'] = True
 
+        nano_cpus = None
+        if 'cpus' in options:
+            nano_cpus = options.get('cpus') * 1000000000
+            if isinstance(nano_cpus, float) and not nano_cpus.is_integer():
```
Comment (shin-) [inline compose/service.py]: I'd really like us to do that check in `validation.py` so it can be caught when running `docker-compose config`

**Comment by AlexeyRokhin** [comment]
Thank you for your time!

**Pull request merged**